### DISCUSSION TITLE
When we are nesting rows/columns, why is the uppermost level not displayed inside a card?

### DISCUSSION THREAD

**Discussion by Anindya088** [zauberzeug/nicegui#2696] [Q&A] [answered]
### Question

Hi All,

I was trying to understand how to nest rows and columns inside each other. Usually I just do that with a ```with``` condition and they work perfectly fine. 

But I realized if I am doing this inside a card, then the uppermost level is skipped. Below is an example of what I mean.
Can you please help me understand why is this happening? Is it expected to happen like this?

```python
ui.label('We try 3 level nesting')
with ui.row().classes('border border-black w-[300px] h-[200px]'):
    with ui.column().classes('border border-black w-[100px] h-[100px]'):
        ui.row().classes('border border-black w-[50px] h-[50px]')
```
![image](https://github.com/zauberzeug/nicegui/assets/65127974/7fbb291e-e549-4146-b1a8-92a635eeca92)

```python
ui.label('First level is skipped, level 2 and 3 is displayed')
with ui.card().classes('border border-black w-[400px] h-[300px]'):
    with ui.row().classes('border border-black w-[300px] h-[200px]'):
        with ui.column().classes('border border-black w-[100px] h-[100px]'):
            ui.row().classes('border border-black w-[50px] h-[50px]')
```
![image](https://github.com/zauberzeug/nicegui/assets/65127974/3d269485-141e-4409-9f2e-bbb3065c0aa1)

```python
ui.label('I repeat the first level 2 times in code, then it comes up')
with ui.card().classes('border border-black w-[400px] h-[300px]'):
    with ui.row().classes('border border-black w-[300px] h-[200px]'):
        with ui.row().classes('border border-black w-[300px] h-[200px]'):
            with ui.column().classes('border border-black w-[100px] h-[100px]'):
                ui.row().classes('border border-black w-[50px] h-[50px]')
```
![image](https://github.com/zauberzeug/nicegui/assets/65127974/aa4eb9d5-ebb2-42dd-a0a9-e9b7069dc869)

Thanks a lot,
Anindya

**Comment by falkoschindler** [comment, marked as answer]
Hi @Anindya088, this is a known issue and we're already working on a solution: https://github.com/zauberzeug/nicegui/issues/2265
But since we'll probably need to break the existing API a bit, we're waiting for the next major release.

**Reply by Anindya088** [reply]
Thanks a lot @falkoschindler for the update.
